app/views/tag_assignations/index.html.erb

```diff
@@ -1,3 +1,3 @@
-<%= render "common/remote_modal", title: "Choisissez un tag", auto_dismiss: false do %>
-  <%= render "form", tags: @available_tags %>
+<%= render "common/remote_modal", title: "Choisissez un ou plusieurs tag(s)", auto_dismiss: true do %>
+  <%= render "form", tags: @available_tags - @user_tags %>
```

Comment:
C'était pas précisé sur les maquettes mais on est sûr qu'on veut pas voir les tags déjà assignés

Reply:
Alors j'ai essayé les 2 (avec et sans) mais c'est pas cohérent de les garder car la modal c'est réellement "Ajouter un tag" et si on les garde cela veut dire qu'on permet de les décocher donc c'est implicite mais ça voudrait dire qu'on pourrait les supprimer. Et vu qu'on a un workflow dédié à la suppression c'est pas clair. 

Si on veut les afficher il faut selon moi les disable pour garder le workflow de suppression. Mais ça me semble inutile. 

Pour moi le workflow optimal c'est celui fait ici c'est à dire avec 2 workflows clairs, un pour ajouter et un pour supprimer, où ajouter un tag ne permet que d'ajouter des nouveaux tags

Reply:
Moi ça me va je ping juste @samanthaauffret pour qu'elle soit au courant

Reply:
Effectivement le ticket github ne précisait pas le cas d'un usager qui a déjà des tags de renseignés...
 
Ok. Du coup si on fait les deux workflows clairs comme dit @Michaelvilleneuve, il faudrait à mon sens optimiser la modale d'ajout des tags avec les tags déjà renseignés. Durant cette action d'ajout, l'utilisateur ne voit plus les tags déjà renseignés. 
Je peux faire l'update de la maquette de la popin. ça va concerner cette modale d'ajout dans la fiche usager mais également la modale d'édition des tags dans la fonctionnalité "upload" (qui est prête pour le prochain sprint). 

Ok pour vous ?

Reply:
Ok j'ai pas bien compris ce que tu voulais changer mais j'attends ton update sur la maquette ^^

Reply:
Voici les maquettes mises à jour de la popin :  https://www.figma.com/design/ngUPYLSKMq1ttDl0sJHQCF/Dossier-Usager?node-id=351-140&t=weDGsNn3xoM8A0Fx-4

Ce qui change vis-à-vis de la précédente version :  
- Titre de la popin : "choisissez" devient "ajouter" + "à l'usager" à la fin du titre
- Le CTA "Valider" devient "ajouter" + ajout d'un CTA "annuler"
- Dans le cas d'une première sélection de tag, on garde les checkbox, il y a juste le titre et les CTA qui changent comme énoncé ci-dessus
- Dans le cas d'une sélection de tag complémentaire à une première sélection, on affiche les tags qui avaient déjà été renseignés + la liste en checkbox des autres tags pouvant être ajoutés (cf. lien maquette). 

Du coup on a bien deux workflows : 1 pour la suppression et 1 pour l'ajout de tag. 

Est-ce que ça vous va @aminedhobb et @Michaelvilleneuve ? 
A dispo pour un petit point si nécessaire

Reply:
J'ai mis à jour la PR @aminedhobb, j'ai également mis les visuels à jour dans la description

Reply:
@Michaelvilleneuve merci pour l'update. j'ai des retours vis-à-vis de cette nouvelle version : 
- pourquoi les CTA "retirer" et "supprimer" n'ont pas les icônes qui sont proposés sur les maquettes, prb technique  ? 
- Remplacer dans le titre, le terme "choisissez" par "ajouter" qui serait plus correct avec le CTA
- Pour la pop-in illustrant le cas "Aucun autre tag disponible", je pense qu'il faut garder le style "empty" qu'on a sur la page : garder le sous-titre "ajouter des tags" et mettre la phrase en bleu clair "Aucun autre tag disponible dans cette organisation" (car la majorité des utilisateurs de rdvi n'ont pas accès à la fonctionnalité de configuration de l'organisation). As-tu besoin d'une maquette pour illustrer ce dernier point si c'est pas très clair ?

Reply:
@samanthaauffret En fait on se base sur la police d'icone suivante : https://fontawesome.com/ et on se limite donc volontairement à ce qu'elle propose pour des raisons de cohérence

Pour le second point c'est noté je m'en charge ! 
Pour le 3ème je veux bien histoire d'être sur d'avoir bien compris ce que tu souhaites, merci !

Reply:
@Michaelvilleneuve , 
Au sujet des pictos et de la cohérence globale, je pense qu'il faudra prévoir un chantier de maj des icones rdvi pour passer de fontawesome à ceux de la librairie du GIP. 
En attendant, tu peux utiliser pour le CTA "retirer" ce picto https://fontawesome.com/icons/minus?f=classic&s=solid et pour le cta "supprimer" ce picto https://fontawesome.com/icons/trash-can?f=classic&s=solid 

Et voilà la maquette pour le dernier point : https://www.figma.com/design/ngUPYLSKMq1ttDl0sJHQCF/Dossier-Usager?node-id=385-141&t=XT7E4G96yyBSsQ8X-4